src/ViscousEvolution.py

```diff
@@ -137,32 +147,22 @@ class LBP_Solution(object):
         n_c   : viscosity evaluated at R = rc
         gamma : radial dependence of nu, default = 1 (viscosity power law index)
         
-    tc : Viscous timescale, commonly known as t_{\nu}
-    Sigma0 : Constant factor in the gas surface density
+    t_c : Viscous timescale, commonly known as t_{\nu}

```

Comment:
Ok but the underscore I meant at the beginning :) (i.e. they should be _tc and _Sigma0)